static/templates/settings/realm-logo-widget.hbs

```diff
@@ -1,4 +1,5 @@
 <div id="{{theme_mode}}-logo-section" class="realm-logo-section">
+    <h5>{{#if (eq theme_mode 'day')}} Day {{else}} Night {{/if}} logo</h5>
```

Comment:
This logic doesn't worth i18n; we should just have the whole strings inside the if/else.

https://zulip.readthedocs.io/en/latest/translating/internationalization.html is good reading for why (different languages have a different word order, so you can't concatenate strings this way).

Reply:
ok